nva-datamodel-java/src/test/java/no/unit/nva/PublicationContextTest.java

```diff
@@ -82,10 +82,12 @@ void shouldWriteEveryNvaClassAsFragmentUri() {
     }
 
     @Test
-    void shouldDefineEveryArrayPropertyAsContainerSet() {
```

Comment:
This test was weird, I changed  it so it looks only at the definitions in the model without regard to having 100% equality with the context object